### PR TITLE
About us

### PR DESCRIPTION
**Pull request by janegreene**
### New Feature Submissions:
* [x] Have you added an explanation of what your changes do?
* [x] Have you linted your code locally prior to submission?

### Testing:
* [ na] Have you written new tests for your core changes, as applicable?
* [x ] Have you successfully ran tests with your changes locally?
* [ x] All new and existing tests passed?

**Test coverage** =99%
--------------------------------------------------------------------------------
### Change Log:
Added link in nav bar and route for about page
Added base for About Us page, needs more flavor.

### CODE REVIEW

File: app/views/developers/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        <a href="https://www.linkedin.com/in/kathy-bui-87a27a1ab/">LinkedIn</a><a> | </a>
+        <a href="https://github.com/Kathybui732">GitHub</a>
+      </div>
+
+      <div class="gallery center-search">
+        <a target="_blank" >
+          <img src="https://avatars2.githubusercontent.com/u/60492885?s=460&u=a23ddc09c48644913428c412632c5535ce5cb703&v=4" alt="Dorion" width="600" height="400">
+        </a>
+        <br>
+        <p>Rion H</p>
```
Comment (janegreene) [inline app/views/developers/index.html.erb]: @sciencefixion how do you want your name to look?

**Review by janegreene — COMMENTED** [review]
Would love to use the animation JS on this page if y'all think that would be cool?

File: app/views/developers/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+      <div class="gallery center-search">
+        <a target="_blank" >
+          <img src= "https://avatars3.githubusercontent.com/u/62079009?s=460&u=ba54507a91cc41193ffa757e661cfc5e708370a7&v=4" alt="Dan Sehl" width="600" height="400">
+        </a>
+        <br>
+        <p>Dan Sehl</p>
+        <a href="https://www.linkedin.com/in/danielsehl/">LinkedIn</a><a> | </a>
+        <a href="https://github.com/dtsehl">GitHub</a>
+      </div>
```
Comment (kathytbui) [inline app/views/developers/index.html.erb:85]: It looks so good!!!

**Comment by dtsehl** [comment]
Looks great! Levels up how our app looks!

File: app/views/developers/index.html.erb (hunk 1 above)
Comment (sciencefixion) [reply app/views/developers/index.html.erb]: Just my first name is fine :)

**Review by sciencefixion — APPROVED** [review]
This is beautiful. Thank you so much!

**Pull request merged**